mcs/tools/mdoc/Test/DocTest-enumerations.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using ObjCRuntime;
+namespace ObjCRuntime {
+	[Flags]
+	public enum Platform : ulong {
+		None = 0,
+		iOS_2_0 = 131072,
```

Comment:
Please use *hex* values here, to make it easier to figure out how things line up, bitmask-wise.